src/components/SingleArticleView.jsx

```diff
@@ -1,14 +1,14 @@
 import { useEffect, useState } from "react";
 import { Link, useParams } from "react-router-dom";
-import { getArticleById } from "../../utils/api";
+import { getArticleById, getArticles, updateArticleVote } from "../../utils/api";
 import { CommentsView } from "./CommentsView";
 
 export const SingleArticleView = () => {
   const [singleArticle, setSingleArticle] = useState({});
   const [displayComments, setDisplayComments] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [isError, setIsError] = useState(false);
-
+  const [articles, setArticles] = useState([])
```

Comment:
This isn't being used anywhere, unused variables need tidying up as in production this can cause your deploy to fail as they are seen as errors.